architecture/backend/database-sceme/language-testing.sql

```diff
@@ -0,0 +1,93 @@
+drop database if exists language_testing;
```

Comment:
Please, add the MySQL Workbench model with ER Diagram here. It will fully represent the data much more clearly, than the SQL script.
At the same time, SQL script can be easily generated from the model